acceptance-tests/runner/.gitignore

```diff
@@ -0,0 +1,23 @@
+#Ignore all dot files
```

Comment:
not sure if this file works - can you have multiple gitignores? (as there is on at the root for blue ocean)